NSS-Day-Cohort-69/tabloid-one-more-div
# Md feature/4/list post by category

### **Purpose**
To allow a user to filter what post they see by category

### **Files Changed**
- Added useStates to set the categories, and categoryId in `PostList.jsx`
- Added a useEffect to get all the categories in `PostList.jsx`
- Added a filter to filter the post if a category is selected in `PostList.jsx`
- Added the Input field with the categories mapped in the Options in `PostList.jsx`

### **To Test**

Fetch, setup, and run according to project README, then confirm the following

- Logged in user can click the dropdown and select a category
- Logged in user gets a updated view of the corresponding post with the selected category

closes #4
